nav2_behavior_tree/include/nav2_behavior_tree/bt_action_node.hpp

```diff
@@ -257,7 +289,7 @@ class BtActionNode : public BT::ActionNodeBase
   /**
    * @brief Function to send new goal to action server
    */
-  void on_new_goal_received()
```

Comment:
Please rename this function back to `on_new_goal_received`, to be in the styling of the rest of the file

Reply:
Definitely. Though, just to confirm; this is a protected member function so it's not part of the API and probably doesn't need to follow the `on_` method name styling. I changed the name since now this function is only responsible for sending a new goal, whereas previously it was responsible for handling the complete new goal behavior. Should I revert it back to `on_new_goal_received`?

Reply:
That is a good point, I thought that was something overridable. I'd personally prefer to keep it the same name but I think your suggestion is also valid